intents/intent.ridl

```diff
@@ -0,0 +1,183 @@
+webrpc = v1
+
+name = sequence-waas-intents
+version = v0.1.0
+
+struct Intent
+  - version: string
+  - name: string
+  - expiresAt: uint64
+  - issuedAt: uint64
+  - data: any
+  - signatures: []Signature
+    + go.tag.json = ,omitempty
```

Comment:
`,omitempty` is valid, ya? idea is to just have it be omitted.. or what..?

or do you want... `signatures,omitempty` ?

Reply:
It is valid. It means use default name and omit field when empty.

Reply:
hey buddy, so I'm away how json struct tags work and `,omitempty` works -- however, your code is incorrect because its generating the code..

```go
type Intent struct {
	Version    string       `json:"version"`
	Name       string       `json:"name"`
	ExpiresAt  uint64       `json:"expiresAt"`
	IssuedAt   uint64       `json:"issuedAt"`
	Data       interface{}  `json:"data"`
	Signatures []*Signature `json:",omitempty"`
}
```

What are you trying to achieve...? Are you trying to either:
1. Make sure the field is completely omitted from json encoding, in which case use `json:"-"`
2. Are you trying to have it be omitted if the slice is nil, but included if it is non-nil? in which case, this should be: `json:"signatures,omitempty"`

so.. depending what your'e trying to design for, you need to change the RIDL to 

either

```
- signatures: []Signature
    + go.tag.json = -
```
or
```
- signatures: []Signature
    + go.tag.json = signatures,omitempty
```